src/components/Alerts/__test__/Alerts.test.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect } from "react"
+import {
+  render,
+  cleanup,
+  fireEvent,
+  act,
+  wait,
+  waitForDomChange,
+} from "react-testing-library"
+import "jest-styled-components"
+import "jest-dom/extend-expect"
+import * as theme from "../../../common/styles/variables"
```

Comment:
Use the `ThemeProvider` already configured with knitui themes.
`import { ThemeProvider } from "../../../common/styles"`